aiogram/webhook/aiohttp_server.py

```diff
@@ -198,16 +204,27 @@ class SimpleRequestHandler(BaseRequestHandler):
     """
 
     def __init__(
-        self, dispatcher: Dispatcher, bot: Bot, handle_in_background: bool = True, **data: Any
+        self, dispatcher: Dispatcher,
+        bot: Bot,
+        handle_in_background: bool = True,
+        secret_token: Optional[str] = None,
+        **data: Any
     ) -> None:
         """
         :param dispatcher: instance of :class:`aiogram.dispatcher.dispatcher.Dispatcher`
         :param handle_in_background: immediately respond to the Telegram instead of
             waiting end of handler process
         :param bot: instance of :class:`aiogram.client.bot.Bot`
         """
-        super().__init__(dispatcher=dispatcher, handle_in_background=handle_in_background, **data)
+        super().__init__(
+            dispatcher=dispatcher,
+            handle_in_background=handle_in_background,
+            **data)
         self.bot = bot
+        self.secret_token = secret_token
+
+    def verify_secret(self, telegram_secret_token: str) -> bool:
+        return telegram_secret_token == self.secret_token
```

Comment:
It's a bad practice to compare secrets via `==`. Use [compare digest](https://docs.python.org/3/library/secrets.html#secrets.compare_digest)